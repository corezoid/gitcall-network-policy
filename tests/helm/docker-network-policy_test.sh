#!/usr/bin/env bash
$CZ helm template | sed -n '/docker-network-policy\.yaml/,/Source\:/p'
