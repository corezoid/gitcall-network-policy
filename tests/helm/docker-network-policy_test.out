# Source: gitcall_network_policy_release/charts/gitcall_network_policy/templates/docker-network-policy.yaml
apiVersion: crd.projectcalico.org/v1
kind: NetworkPolicy
metadata:
  name: RELEASE-NAME-gitcall-docker-network-policy
  namespace: "aNamespace"
  labels:
    release: "RELEASE-NAME"
    app: "gitcall"
    tier: "network_policy"
spec:
  selector: tier == 'docker_server'
  types:
    - Egress
    - Ingress
  ingress:
    # Prometheus
    - action: Allow
      protocol: TCP
      source:
        namespaceSelector: projectcalico.org/name == 'monitoring'
        selector: app == 'prometheus'
      destination:
        ports:
          - 9323
    # Gitcall
    - action: Allow
      protocol: TCP
      source:
        selector: tier == 'gitcall'
      destination:
        ports:
          # TLS
          - 2376
          # Plain
          - 2375
  egress:
    - action: Allow
---
# Source: gitcall_network_policy_release/charts/gitcall_network_policy/templates/docker-registry-network-policy.yaml
